Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -1013,6 +1013,10 @@ void findDoubleBonds(
           if (nbr == oatm) {
             continue;
           }
+          auto obnd = mol.getBondBetweenAtoms(atm->getIdx(), nbr->getIdx());
+          if (obnd->getBondType() != Bond::BondType::SINGLE) {
```

Comment:
```suggestion
          const auto obnd = mol.getBondBetweenAtoms(atm->getIdx(), nbr->getIdx());
          if (!obnd || obnd->getBondType() != Bond::BondType::SINGLE) {
```
Probably `obnd` can never be `nullptr` so this is unnecessary; feel free to ignore my suggestion if this is the case.

Reply:
yeah, it can't be nullptr, but checking is still not a bad thing